extensions/objects/templates/coord-two-dim-editor.component.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
*2021

Reply:
I don't think we are supposed to change that. +cc @DubeySandeep should we send a mail about this?